### PR TITLE
Updating client library to 2019-05-29 API version

### PR DESCRIPTION
**Pull request by jasonjia93**
Updating INTL and investment API responses based on the changelog: https://docs.google.com/document/d/1mhqceql2JXUw8h2Tak_4lP3pzj8jsDSNw5rp1hksIEU/edit#

### CODE REVIEW

File: src/main/java/com/plaid/client/response/IdentityGetResponse.java [hunk 1]
```diff
@@ -3,77 +3,56 @@
...
 
-  public Identity getIdentity() {
-    return identity;
+  public List<AccountWithOwners> getAccounts() {
+    return accounts;
   }
 
-  public List<Account> getAccounts() {
-    return accounts;
+  public static final class AccountWithOwners extends Account {
```
Comment (ntindall) [inline src/main/java/com/plaid/client/response/IdentityGetResponse.java]: I don't think this should be static, but I am not 100% sure.

@davidzhanghp

Comment (jasonjia93) [reply src/main/java/com/plaid/client/response/IdentityGetResponse.java]: He is in china :(

Comment (jasonjia93) [reply src/main/java/com/plaid/client/response/IdentityGetResponse.java]: Also looking at the other responses, all of the classes are static

Comment (ntindall) [reply src/main/java/com/plaid/client/response/IdentityGetResponse.java]: 👍 sounds good.

Comment (nesabjafar) [reply src/main/java/com/plaid/client/response/IdentityGetResponse.java]: TransactionsGetResponse.java Is this have any fields of  
1- checkNumber
2- routingNumber
3-accountNumber
  if yes 
please let me know

**Pull request merged**